question.py

```diff
@@ -34,4 +34,13 @@ def __valid_answer(self, s):
                 return None
 
         return s
-        
\ No newline at end of file
+
+    def add_sentence(self, sentence):
+        self.sentence = sentence
+
+    def entities(self):
+        entities = []
+        for span in self.sentence._.children:
```

Comment:
sentence._.children will only get the highest-level constituent children from the sentence; we should either do this recursively to add an entity for every lowest-level constituent or iterate over all of the tokens in the sentence.

I think recursively is probably the best way; I'll go ahead and change it.